embed/serve.go

```diff
@@ -71,6 +73,7 @@ func (sctx *serveCtx) serve(s *etcdserver.EtcdServer, tlscfg *tls.Config, handle
 
 	if sctx.insecure {
 		gs := v3rpc.Server(s, nil)
+		v3electionpb.RegisterElectionServer(gs, v3election.NewElectionServer(v3client.New(s)))
```

Comment:
it doesn't really make a difference since it's all pass-through, I wanted to save a line of code